README.md

```diff
@@ -144,6 +144,7 @@ The Hardhat Chainlink plugin supports the following Chainlink services:
 - `automationRegistry` (Automation Registries)
```

Comment:
Recommend we move this Available Services section up -- this should be before "USAGE" so that usage then makes sense in the context of available services that are listed.